### PR TITLE
Copyedits to FFI and unwinding section

### PR DESCRIPTION
**Pull request by tmandry**
These are minor changes that make the text flow better to me.

cc #365

### CODE REVIEW

File: src/ffi.md [hunk 1]
```diff
@@ -722,16 +722,19 @@ No `transmute` required!
 ## FFI and unwinding
 
 It’s important to be mindful of unwinding when working with FFI. Each
-non-`Rust` ABI comes in two variants, one with `-unwind` suffix and one without. If
-you expect Rust `panic`s or foreign (e.g. C++) exceptions to cross an FFI
-boundary, that boundary must use the appropriate `-unwind` ABI string (note
-that compiling with `panic=abort` will still cause `panic!` to immediately
-abort the process, regardless of which ABI is specified by the function that
-`panic`s).
+ABI comes in two variants, one with `-unwind` suffix and one without (except
```
Comment (tmandry) [inline src/ffi.md:725]: Wait, I'm not sure this is always true. Surely there must be some ABIs without a well-defined unwind behavior?

cc @BatmanAoD

Comment (BatmanAoD) [reply src/ffi.md:725]: This bit is talking about `panic!` the macro, not actual stack unwinding. It always aborts when the panic mode is `abort`.

(unless I am misunderstanding the question...)

Comment (tmandry) [reply src/ffi.md:725]: Sorry, I'm referring to the idea that every non-Rust ABI has an -unwind variant. It seems totally reasonable for some ABIs not to have a native unwinding mechanism.

Comment (ehuss) [reply src/ffi.md:725]: There are a few listed [here](https://github.com/rust-lang/rust/blob/4b51adf6ffa1ae2286ea436eb48cbaa8771886e6/compiler/rustc_target/src/spec/abi.rs#L94-L109) which don't have `-unwind` variants.  All of them are unstable/experimental. I would just change it from "Each ABI" to "Most ABIs".

Comment (BatmanAoD) [reply src/ffi.md:725]: That makes sense, and I'll make the change; though I'm not aware of any reason inherent to the ABI definitions themselves that would make `unwind` inappropriate.

**Comment by BatmanAoD** [comment]
Who needs to review this?

**Pull request merged**